cgmanifest.json

```diff
@@ -0,0 +1,153 @@
+{"Registrations":[ 
+     {
+      "Component": {
+         "Type": "Maven",
+         "Maven": {
+           "ArtifactId": "gradle",
+           "GroupId": "com.android.tools.build",
+           "Version": "3.3.2"
+         }
+       }
+    },
+    {
+        "Component": {
```

Comment:
Why do we use not lower case here? Let's harmonize all usages

Reply:
Fixed